src/test/java/ca/corefacility/bioinformatics/irida/web/controller/test/unit/AnnouncementsControllerTest.java

```diff
@@ -125,27 +117,22 @@ public void testGetControlCentreAdminPage() {
         assertTrue("Unexpected page returned", ANNOUNCEMENT_ADMIN.equals(page));
     }
 
-    @Test
-    public void testGetCreateAnnouncementPage() {
-        assertTrue("Unexpected page returned", ANNOUNCEMENT_CREATE.equals(announcementsController.getCreateAnnouncementPage()));
-    }
-
     @Test
     public void testSubmitCreateAnnouncement() {
-        Principal principal = () -> USER_NAME;
-        ExtendedModelMap model = new ExtendedModelMap();
-
-        String message = "The newest message";
-        Announcement a = new Announcement(message, user);
-
-        when(announcementService.create(any(Announcement.class))).thenReturn(a);
-        when(userService.getUserByUsername(USER_NAME)).thenReturn(user);
-
-        announcementsController.submitCreateAnnouncement(message, model, principal);
-
-        assertTrue("Unexpected page returned", ANNOUNCEMENT_CREATE.equals(announcementsController.getCreateAnnouncementPage()));
-        verify(announcementService).create(any(Announcement.class));
-        verify(userService, times(1)).getUserByUsername(USER_NAME);
+//        Principal principal = () -> USER_NAME;
```

Comment:
Any specific reason of why this was just commented out?

Reply:
Updated testing in 3659206ab158c19274848ddf022ec21fe458d4ba